test/stats_test.dart

```diff
@@ -0,0 +1,238 @@
+import 'dart:io';
+
+import 'package:hive/hive.dart';
+import 'package:data_offloading_app/logic/stats.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
```

Comment:
add a comment saying what this test class is for like "class for testing statistics page" or smth more in detail and for each test case as well